shrike-examples/contoso/utils/aml_utils.py

```diff
@@ -0,0 +1,234 @@
+"""
```

Comment:
The probe module relies on this file. For instance, it uses aml_run_attach(), and I do not see that one anywhere in shrike. I imagine we could do some refactoring to get rid of some (all?) of these util files, but that felt out of the scope of the current work. If you feel strongly about this we can discuss and I can spend another cycle, but since the users are not supposed to modify anything about the probe component and just use it as-is, I thought it would be OK to not be super optimal here.

Reply:
Yes it's totally fine to leave as-is. I agree with you that it's out of this problem set's scope.